src/Examples/UnitModels/Reactors/plug_flow_reactor_testing.ipynb

```diff
@@ -0,0 +1,1160 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m = ConcreteModel()\n",
+    "m.fs = FlowsheetBlock(default={\"dynamic\": False})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now need to add the property packages to the flowsheet. Unlike the basic [Flash unit model example](http://localhost:8888/notebooks/GitHub/examples-pse/src/Tutorials/Basics/flash_unit_solution_testing.ipynb), where we only had a thermophysical property package, for this flowsheet we will also need to add a reaction property package. We will use the Generic Property and Generic Reaction Frameworks; more information may be found on these methods at https://idaes-pse.readthedocs.io/en/1.8.0/user_guide/components/property_package/index.html."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "m.fs.thermo_params = GenericParameterBlock(default=thermo_props.config_dict)\n",
+    "m.fs.reaction_params = GenericReactionParameterBlock(default={\"property_package\": m.fs.thermo_params,\n",
+    "                                                              **reaction_props.config_dict})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding Unit Models\n",
+    "\n",
+    "Let us start adding the unit models we have imported to the flowsheet. Here, we are adding a Mixer (assigned a name M101), a Heater (assigned a name H101) and a PFR (assigned a name R101). Note that all unit models need to be given a property package argument. In addition to that, there are several arguments depending on the unit model, please refer to the documentation for more details (https://idaes-pse.readthedocs.io/en/latest/model_libraries/core_lib/unit_models/index.html). For example, the Mixer unit model here is given a `list` consisting of names to the two inlets.\n",
+    "\n",
+    "For the PFR, a few optional arguments relate to the spatial discretization. They are taken as default values (20 elements using backwards finite difference) if not provided."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "m.fs.M101 = Mixer(default={\"property_package\": m.fs.thermo_params,\n",
+    "                           \"inlet_list\": [\"reagent_feed\", \"catalyst_feed\"]})\n",
+    "m.fs.H101 = Heater(default={\"property_package\": m.fs.thermo_params,\n",
+    "                            \"has_pressure_change\": False,\n",
+    "                            \"has_phase_equilibrium\": False})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "m.fs.R101 = PFR(\n",
+    "            default={\"property_package\": m.fs.thermo_params,\n",
```

Comment:
I will show how to set the number of elements here (even if you use the default value) - it is important for users to know how to do this. We should also show how to set the discretization method (both arguments).

Reply:
I will add these arguments back in.